doc/INSTALL.md

```diff
@@ -111,7 +109,49 @@ Please refer to the section OS Independent below.
 
 First follow the steps in [COMPILE](COMPILE.md).
 
-After you completed building Elektra on your own, there are multiple options how to install it. For example, with make or cPack tools.
+After you completed building Elektra on your own, there are multiple options how to install it. For example, with make or CPack tools.
+We recommend that you generate your own packages with CPack so ensure compatibility with future releases.
+
+### CPack
+
+The current supported systems are: Debian, Ubuntu and Fedora.
+
+First follow the steps in [COMPILE](COMPILE.md).
+
+Then use:
+
+```sh
+make package
+```
+
+which will create a package for distributions where a Generator is implemented.
+
+You can find the generated packages in the `packages` directory of the build directory.
```

Comment:
Maybe we can rename this folder to "package" as, at least zsh, always suggests to correct package to packages.